server/src/test/java/org/opensearch/gateway/remote/RemoteClusterStateServiceTests.java

```diff
@@ -146,6 +151,57 @@ public void testWriteFullMetadataSuccess() throws IOException {
         assertThat(manifest.getStateUUID(), is(expectedManifest.getStateUUID()));
     }
 
+    public void testWriteFullMetadataAsyncSuccess() throws IOException {
```

Comment:
minor testWriteFullMetadataInParallelSuccess , async can create confusion as if it is uploaded in async fashion. Update the other test name as well.